src/development/tools/devtools/performance.md

```diff
@@ -220,8 +220,14 @@ performance page. To import a performance snapshot, you can drag and drop the
 snapshot into DevTools from any page. **Note that DevTools only
 supports importing files that were originally exported from DevTools.**
 
+## Other resources
+
+To learn how to monitor an app's performance and
+detect jank using DevTools, check out a guided
+[Performance View tutorial][performance-tutorial].
```

Comment:
Really cool articles!

I think the build is failing due to these Markdown link references not resolving. An extra line should fix that:

```suggestion
[Performance View tutorial][performance-tutorial].

```

Reply:
Thank a lot @parlough 🙂
Also, thank you for suggesting the fix. I was wondering why the link check failed.